src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -106,9 +129,17 @@ object Scope {
   }
 
   @tailrec
-  def fullyQualifiedNameBuilder(scope: Option[Scope], fqn: String, fullDisplay: Boolean): String = scope match {
-    case None => fqn.tail //Strip away the first "." of the name
-    case Some(x: Scope) => fullyQualifiedNameBuilder(x.parent, (if (fullDisplay || x.appearsInFQN) s".${x.name}" else "") + fqn, fullDisplay)
+  def fullyQualifiedNameBuilder(scope: Option[Scope], fqn: String, fullDisplay: Boolean, leaf: Boolean): String = {
+    scope match {
+      case None =>
+        fqn.tail //Strip away the first "." of the name
```

Comment:
Can you move this up a line?

Also on the order of Some/None I think it's a little more clear to put the Some first unless there's a good reason not to (just cuz that's how they usually are)